elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcSubsystemParser_3_0.java

```diff
@@ -34,4 +34,65 @@ public class ElytronOidcSubsystemParser_3_0 extends ElytronOidcSubsystemParser_2
     String getNameSpace() {
         return NAMESPACE_3_0;
     }
+    
+    final PersistentResourceXMLDescription secureDeploymentParser = secureDeploymentParserBuilder
```

Comment:
Looks like the parser changes are now showing up as part of the subsystem bump commit.

Reply:
fixed